modules/ROOT/pages/shared-resources.adoc

```diff
@@ -17,31 +18,30 @@ To share the metadata, keep the Mule domain project open in Anypoint Studio. Oth
 
 == Prerequisites
 
-This document assumes that you are using Anypoint Studio 7 with Mule runtime 4, or that you are building your apps using Studio 7 and deploying them to Mule 4 Standalone runtime.
+* Anypoint Studio 7.x
+* Mule runtime engine 4.x
 
 == Limitations
 
 Defining flows, subflows, or any message processors as shared resources is not supported. Domains are not meant to share behavior. They are meant to share resources only.
 
-== Basic Anatomy
+== Architecture
 
-=== Studio Project
+A Mule domain Studio project has the following key files:
 
-The following files are key within the Mule Domain Project:
-
-[%autowidth.spread]
+[%header%autowidth.spread,cols="a,a"]
 |===
+|File name |Description
 |`/src/main/mule/mule-domain-config.xml` |This is the shared resources configuration file. This file must have the `mule-domain-config.xml` file name.
 |`pom.xml` |This is the dependencies file descriptor. It is a Maven POM file with required plugins for domains projects.
 |`/mule-artifact.json` |This file is a descriptor for the domain artifact.
 |===
 
-=== Domain Artifact
-
-You can find the following files when you package a domain into a JAR file:
+When you package the domain into a JAR file, you can find the following files:
```

Comment:
Is packaging the domain into a JAR file what creates the Mule domain project?
Again, I am wondering if the earlier text should specify this: "...you create a Mule domain project..." > "you use Anypoint Studio to package the domain into a JAR file, creating a Mule domain project..."?